pkg/util/pflag_import.go

```diff
@@ -85,23 +85,25 @@ func addFlagToPFlagSet(f *flag.Flag, fs *pflag.FlagSet) {
 	}
 }
 
-// Adds all of the flags in a 'flag.FlagSet' package flags to a 'pflag.FlagSet'.
+// AddFlagSetToPFlagSet adds all of the flags in a 'flag.FlagSet' package flags to a 'pflag.FlagSet'.
 func AddFlagSetToPFlagSet(fsIn *flag.FlagSet, fsOut *pflag.FlagSet) {
 	fsIn.VisitAll(func(f *flag.Flag) {
 		addFlagToPFlagSet(f, fsOut)
 	})
 }
 
-// Add all of the top level 'flag' package flags to the top level 'pflag' flags.
+// AddAllFlagsToPFlags adds all of the top level 'flag' package flags to the top level 'pflag' flags.
 func AddAllFlagsToPFlags() {
 	AddFlagSetToPFlagSet(flag.CommandLine, pflag.CommandLine)
 }
```

Comment:
Actually I was also trying to get rid of those funcs there. The problem is that cobra is merging flag command line flags but is not working for the pflag global flags. Also an other problem is the order and "when" each of those merges are taking place. This is actually an issue of cobra and I might work on that at some point: spf13/cobra#44